readme.md

```diff
@@ -100,6 +100,13 @@ Default: `false`
 
 Show the `Copy Image Address` menu item when right-clicking on an image.
 
+#### showCopyImage
```

Comment:
Not the correct order. The order here should follow the order shown in the menu.